sdk/keyvault/keyvault-secrets/review/keyvault-secrets.api.md

```diff
@@ -4,34 +4,54 @@
 
 ```ts
 
+import * as coreHttp from '@azure/core-http';
 import { HttpClient } from '@azure/core-http';
 import { HttpPipelineLogger } from '@azure/core-http';
-import * as msRest from '@azure/core-http';
 import { PagedAsyncIterableIterator } from '@azure/core-paging';
 import { PageSettings } from '@azure/core-paging';
+import { Poller } from '@azure/core-lro';
+import { PollOperationState } from '@azure/core-lro';
 import { RequestOptionsBase } from '@azure/core-http';
 import { ServiceClientOptions } from '@azure/core-http';
 import { TokenCredential } from '@azure/core-http';
 
 // @public
-export interface DeletedSecret extends Secret {
-    readonly deletedDate?: Date;
-    recoveryId?: string;
-    readonly scheduledPurgeDate?: Date;
+export interface DeletedSecret {
+    properties: SecretProperties & {
+        recoveryId?: string;
+        readonly scheduledPurgeDate?: Date;
+        readonly deletedDate?: Date;
```

Comment:
`deletedOn`. I'll not repeat for future instances (spam), but please check them all. In fact, all `Date` fields should be suffixed with "On" **except** for `notBefore` and `scheduledPurgeDate`.

Reply:
Addressing this here: https://github.com/Azure/azure-sdk-for-js/issues/5577